pkg/apis/storage/types.go

```diff
@@ -157,7 +157,17 @@ type VolumeAttachmentSource struct {
 	// +optional
 	PersistentVolumeName *string
 
-	// Placeholder for *VolumeSource to accommodate inline volumes in pods.
+	// Represents the source location of a in-line volume in a pod.
+	InlineVolumeSource *InlineVolumeSource
```

Comment:
Did we want to handle this as part @davidz627 's migration changes instead? I think this field would be under the CSIMigration feature gate instead of the CSIInlineVolumes feature gate.

Reply:
Ah yes, I missed that one.  Thanks will remove and repush.